vm/src/sysmodule.rs

```diff
@@ -434,6 +508,15 @@ settrace() -- set the global debug tracing function
       "__displayhook__" => ctx.new_function(sys_displayhook),
     });
 
+    #[cfg(windows)]
+    {
+        let getwindowsversion = WindowsVersion::make_class(ctx);
+        extend_module!(vm, module, {
+            "getwindowsversion" => ctx.new_function(sys_getwindowsversion),
+            "getwindowsversion_type" => getwindowsversion,
```

Comment:
Because this is not a documented or a CPython compitible attribute, naming to `_getwindowsversion_type` to mark it non-public and adding comment like '// XXX: This is not a python spec but required by current RustPython implementation` will be appreciated

Reply:
I just added some test cases and renamed the type. Could you check it again?